server/CONTRIBUTING.md

```diff
@@ -11,11 +11,12 @@ own machine and how to contribute.
 - npm >= 5.7
 - [gsutil](https://cloud.google.com/storage/docs/gsutil)
 - libpq-dev
+- libkrb5-dev
 - python >= 3.5 with pip3
 
-The last two prerequisites can be installed on Debian with:
+The last two prerequisites can be installed on Debian or Ubuntu with:
```

Comment:
Is writing `Ubuntu` necessary, since Ubuntu is Debian based? Any specific reason for adding it ?

Reply:
I think it doesn't hurt to write, and it's possible that someone (especially a newer dev) might not be aware that Debian and Ubuntu packages have the same names. Many more people use Ubuntu than Debian, so I thought it might be worth adding.